Apps/WebClient/src/appsettings.json

```diff
@@ -65,13 +65,12 @@
             "CovidLabResults": true,
             "Dependent": true,
             "MedicationRequest": true,
-            "Credential": false,
             "VaccinationStatus": true,
             "VaccinationStatusPdf": true,
-            "WalletExport": false,
-            "VaccinationExportPdf": false,
-            "PublicVaccineDownloadPdf": false,
-            "FederalCardButton": false
+            "WalletExport": true,
```

Comment:
Saving to the BC Vaccine Wallet is currently disabled in PROD, so we may want to default "WalletExport" to false.

Reply:
Good catches, fixed.